Add docs for toggle_profile_activation command (#763)
Uninett/Argus: docs/development/management-commands.rst

This incident will automatically be closed if the token related to the incident
is renewed, deleted or replaced by a new one.

.. _toggle-profile-activation:

Toggle profile activation
-------------------------
.. warning::

>     You should be careful using the `toggle_profile_activation` command against a

Reviewer: ```suggestion
    You should be careful using the `toggle_profile_activation` command in a
```

    production environment, since activating previously inactive notification profiles

>     can lead to notifications and deactivating previously active profile might lead to

Reviewer: ```suggestion
    can lead to notifications, and deactivating a previously active profile might lead to
```

    missing notifications.

To quickly make a notification profile (in)active one can use the command
`toggle_profile_activation`:

    .. code:: console

        $ python manage.py toggle_profile_activation

See the inbuilt help for flags and toggles:

    .. code:: console

        $ python manage.py toggle_profile_activation --help

> This command takes a list of notification profile ids separated by a space as an
> argument:

Reviewer: ```suggestion
This command takes a list of notification profile ids (separated by spaces) as
arguments:
```
